Formula/rpm.rb

```diff
@@ -27,54 +12,47 @@ class Rpm < Formula
     sha256 "3a276b6a3f7273f8f88effbb057e8d65fc53fa9e812054951bfdd5ce618f54ac" => :yosemite
   end
 
-  depends_on "rpm2cpio" => :build
+  depends_on "pkg-config" => :run
   depends_on "berkeley-db"
+  depends_on "gettext"
+  depends_on "libarchive"
   depends_on "libmagic"
+  depends_on "lua"
+  depends_on "openssl@1.1"
   depends_on "popt"
-  depends_on "libtasn1"
-  depends_on "gettext"
   depends_on "xz"
-  depends_on "ossp-uuid"
+  depends_on "zstd"
 
   def install
     # only rpm should go into HOMEBREW_CELLAR, not rpms built
-    inreplace "macros/macros.in", "@prefix@", HOMEBREW_PREFIX
+    inreplace "macros.in", "@prefix@", HOMEBREW_PREFIX
+    inreplace "platform.in", "@prefix@", HOMEBREW_PREFIX
```

Comment:
Since these are both the same replacement you can do
```
    inreplace ["macros.in", "platform.in"], "@prefix@", HOMEBREW_PREFIX
```

Reply:
Fixed